pype/hooks/nukestudio/pre_launch_args.py

```diff
@@ -0,0 +1,17 @@
+import os
+from pype.lib import PreLaunchHook
+
+
+class NukeStudioLaunchArguments(PreLaunchHook):
+    order = 0
+    app_groups = ["nukestudio"]
+
+    def execute(self):
+        """Prepare suprocess launch arguments for NukeStudio."""
+        # Add path to workfile to arguments
+        if self.data.get("start_last_workfile"):
```

Comment:
Do we set this in some way currently? I assume that logic willing this hook `self.data` is still missing.

Reply:
That is set in global host prelaunch hook `~/hooks/global/pre_global_host_data.py` https://github.com/pypeclub/pype/pull/747/files#diff-cd623e011c6a95b4660d1c8d06aba5d85d39b8c7d84e6dd4583d11bb262c45f2R225